Slambook/index.html

```diff
@@ -1,51 +1,118 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <title>Slam book</title>     
-    <meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
-    <meta name="GENERATOR" content="MSHTML 10.00.9200.16384">
-    <link rel="stylesheet" href="index.css"/>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kalam">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tillana">
-  </head>
```

Comment:
The 'submit' event listener should be attached to the form element (not just the if statement) to ensure it is only added when the form element is found.